website/docs/reference/global-configs.md

```diff
@@ -103,6 +103,36 @@ config:
 
 </File>
 
+<VersionBlock firstVersion="1.1">
+
+### Cache database objects for selected resource
+
+:::caution Experimental config flag
+This should be considered an _unstable_ config flag because it is experimental and subject to change. We reserve the right to make breaking changes to this config flag.
+:::
```

Comment:
This hits on most of the right points for me. This is a thing we will be doing not infrequently, as a way to continue shipping innovative stuff in major-version-1 software. e.g.
- metrics
- https://github.com/dbt-labs/docs.getdbt.com/pull/1301